tests/.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I don't think we should be tracking these - .pytest_cache should be in our .gitignore